blvcktwitter/README.md

```diff
@@ -0,0 +1,47 @@
+# Well ,the simplest way to go about this would be to do it like this...
```

Comment:
The formatting here is off. You can read the following to learn more about open source readmes:

- GitHub’s [About READMEs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes).
- [How to write a good README for your GitHub project?](https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project).

Good readmes are essential to make a project look professional. Take your time reviewing other project readmes and, when you find some time, make a pull request improving this readme 🙂